icons/solidjs/solidjs-plain-wordmark.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><g fill="#518ac8"><path d="M75.81 15.35S50.86-3.49 31.5.91l-1.7.47A12.35 12.35 0 0 0 23.22 6a11.18 11.18 0 0 0-.71 1.18l-7.15 12.15 12.35 2.41a22 22 0 0 0 17.61 3.46l22 4.3z" opacity=".3" style="isolation:isolate" transform="matrix(.38 0 0 .38 -3.464 50.04242)"/><path d="m23 15.11-1.71.48c-7.9 2.55-10.6 9.94-6.06 16.42a21.12 21.12 0 0 0 22.54 7.1l29.52-9.61S42.39 10.71 23 15.11z" opacity=".3" style="isolation:isolate" transform="matrix(.38 0 0 .38 -3.464 50.04242)"/></g><path fill="#518ac8" d="M37.84686 66.987a4.51855 4.51855 0 0 0 3.05003 1.28025c1.28026 0 2.33458-.7531 2.33458-2.03335 0-2.93706-5.83646-2.22162-5.83646-5.83647 0-1.69445 1.50618-3.12533 3.6525-3.12533a4.51855 4.51855 0 0 1 3.12533 1.12964l-.56859.97902a3.90855 3.90855 0 0 0-2.56051-1.01667c-1.50618 0-2.44755.97902-2.4852 1.99569 0 2.78644 5.83646 1.9957 5.83646 5.83646 0 1.73212-1.35557 3.163-3.53953 3.163a5.03066 5.03066 0 0 1-3.69015-1.5062zm18.48842-9.63959a5.92307 5.92307 0 0 1 5.94943 5.98708 6.02474 6.02474 0 0 1-5.94943 6.10005h-.03765a6.02474 6.02474 0 0 1-5.94943-6.10005v-.03766a5.92307 5.92307 0 0 1 5.98708-5.94942zm-.03765 11.0328a4.8951 4.8951 0 0 0 4.78213-5.0043v-.04142c0-2.78644-2.10866-4.93275-4.74448-4.93275a4.78967 4.78967 0 0 0-4.78213 4.8951c0 2.86175 2.10865 5.04572 4.74448 5.08337zm12.7649-10.80687h1.12964l-.03766 10.65625h5.42227v1.01668h-6.58956zm12.76492.03766h1.12964l-.03766 11.71058h-1.12964zm8.73587 0h3.76546c3.50188 0 5.87412 2.14631 5.87412 5.87412 0 3.7278-2.37224 5.83646-5.87412 5.83646h-3.76546zm3.61484 10.6939c2.86175 0 4.78214-1.69445 4.81979-4.81978.03766-3.12534-1.92039-4.8198-4.78214-4.8198h-2.52285l-.03765 9.63959zm15.87895 1.01668a3.86713 3.86713 0 0 0 2.65842-.9489 3.38891 3.38891 0 0 0 1.07692-2.67347v-7.94889h-2.35342V65.526a1.5815 1.5815 0 0 1-.37654 1.12964 1.35933 1.35933 0 0 1-1.04304.4029q-1.10704 0-1.61538-1.29908l-1.99193 1.1108q.97526 2.45132 3.64497 2.45132zm14.10542.02636c1.30661 0 2.33082-.32383 3.06508-.97525a3.24206 3.24206 0 0 0 1.09952-2.55675 2.66971 2.66971 0 0 0-.94137-2.13502c-.6326-.54222-1.64551-1.00914-3.03873-1.40828a4.42818 4.42818 0 0 1-1.41581-.59118.96772.96772 0 0 1-.4142-.80957 1.01667 1.01667 0 0 1 .4142-.84723 1.73211 1.73211 0 0 1 1.09575-.32006 2.13502 2.13502 0 0 1 2.15008 1.40075l1.94674-1.06563a3.78429 3.78429 0 0 0-1.53254-1.82248 4.58257 4.58257 0 0 0-2.45885-.6439 4.18343 4.18343 0 0 0-2.80526.97902 3.05379 3.05379 0 0 0-1.16353 2.46638 2.8241 2.8241 0 0 0 .87358 2.11243q.78322.79074 2.77515 1.38945a6.21678 6.21678 0 0 1 1.70199.68532.87735.87735 0 0 1 .44809.78698 1.03174 1.03174 0 0 1-.48575.86982 2.12372 2.12372 0 0 1-1.2426.34266 2.63582 2.63582 0 0 1-2.4852-1.5928l-2.03335 1.05057a4.08553 4.08553 0 0 0 1.69069 1.96933 5.17374 5.17374 0 0 0 2.75632.71544z" class="g"/></svg>
```

Comment:
It seems that the bottom path of the "s" is missing.
Here's some suggestion of fixing made with Inkscape CLI from `original-wordmark` version.

### Preview
<img alt="preview" width="256" src="https://user-images.githubusercontent.com/76992016/205429295-ee70a4f2-8fc1-4025-9ca9-b8e01a4408e0.svg"/>

> Note: even it's black, actually the path has no fill color, so it'll be filled by `color` property of `devicon.json` entry.

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path d="M40.25 67.017a4.374 4.374 0 0 0 2.952 1.24c1.239 0 2.26-.73 2.26-1.969 0-2.843-5.65-2.15-5.65-5.65 0-1.64 1.458-3.024 3.535-3.024a4.374 4.374 0 0 1 3.026 1.093l-.55.948a3.783 3.783 0 0 0-2.48-.984c-1.457 0-2.368.947-2.405 1.931 0 2.697 5.65 1.932 5.65 5.65 0 1.676-1.313 3.061-3.427 3.061a4.87 4.87 0 0 1-3.571-1.458zm17.895-9.33a5.733 5.733 0 0 1 5.759 5.795 5.832 5.832 0 0 1-5.759 5.904h-.036a5.832 5.832 0 0 1-5.76-5.904v-.037a5.733 5.733 0 0 1 5.796-5.758zm-.036 10.679a4.738 4.738 0 0 0 4.628-4.844v-.04c0-2.697-2.04-4.775-4.592-4.775a4.636 4.636 0 0 0-4.629 4.738c0 2.77 2.041 4.884 4.593 4.92zm12.355-10.46h1.094L71.52 68.22h5.249v.984H70.39zm12.356.036h1.094l-.037 11.335h-1.093zm8.456 0h3.645c3.39 0 5.685 2.077 5.685 5.686 0 3.608-2.296 5.649-5.685 5.649h-3.645zm3.499 10.35c2.77 0 4.629-1.64 4.665-4.664.037-3.026-1.859-4.666-4.629-4.666H92.37l-.036 9.33zm15.37.985a3.743 3.743 0 0 0 2.573-.919 3.28 3.28 0 0 0 1.042-2.587v-7.694h-2.278v7.526a1.53 1.53 0 0 1-.364 1.093 1.316 1.316 0 0 1-1.01.39q-1.071 0-1.563-1.257l-1.928 1.075q.944 2.373 3.528 2.373zm13.653.025c1.265 0 2.256-.313 2.967-.944a3.138 3.138 0 0 0 1.064-2.474 2.584 2.584 0 0 0-.911-2.067c-.612-.525-1.593-.977-2.941-1.363a4.286 4.286 0 0 1-1.37-.572.937.937 0 0 1-.402-.784.984.984 0 0 1 .401-.82 1.677 1.677 0 0 1 1.06-.31 2.067 2.067 0 0 1 2.082 1.356l1.884-1.031a3.663 3.663 0 0 0-1.483-1.765 4.436 4.436 0 0 0-2.38-.623 4.05 4.05 0 0 0-2.715.948 2.956 2.956 0 0 0-1.127 2.387 2.734 2.734 0 0 0 .846 2.045q.758.765 2.686 1.345a6.018 6.018 0 0 1 1.648.663.85.85 0 0 1 .433.762.999.999 0 0 1-.47.842 2.056 2.056 0 0 1-1.203.332 2.551 2.551 0 0 1-2.405-1.542l-1.968 1.017a3.955 3.955 0 0 0 1.636 1.906 5.008 5.008 0 0 0 2.668.692zm-104.36.609a7.705 7.705 0 0 0-8.212-2.606L.46 70.789s9.111 6.867 16.15 5.263l.62-.17c2.882-.916 3.866-3.61 2.208-5.97zm3.382-6.021a7.694 7.694 0 0 0-8.23-2.606L3.842 64.768.46 70.79l19.255 3.28 3.451-6.14c.69-1.178.62-2.661-.346-4.04zM8.646 56.123l-.624.175c-2.879.93-3.863 3.623-2.208 5.985a7.698 7.698 0 0 0 8.215 2.587l10.76-3.502s-9.076-6.849-16.143-5.245zm19.248.087s-9.094-6.866-16.15-5.263l-.62.172a4.501 4.501 0 0 0-2.398 1.684 4.075 4.075 0 0 0-.259.43L5.861 57.66l4.501.878a8.018 8.018 0 0 0 6.419 1.262l8.018 1.567z"/></svg>

```